oaknational/Oak-Web-Application
# [CUR-949] Filter correct pathways / examboards in docx

## Description

Music year: 2014

- Updated the DocX querying to use the same as the visualiser

## How to test

1. Go to https://deploy-preview-2873--oak-web-application.netlify.thenational.academy
2. Click on any programme with pathways
3. You should see the correct pathway or ks4 option selected when downloading the document

## Checklist
